publish/publish.proj

```diff
@@ -11,9 +11,13 @@
   
   <Target Name="PublishFinalOutput"
           Condition="'$(Finalize)' == 'true'"
-          DependsOnTargets="PublishCoreHostPackagesToFeed;FinalizeBuildInAzure;UpdatePublishedVersions" />
-          
-  <Target Name="ExcludeSymbolsPackagesFromPublishedVersions" BeforeTargets="UpdatePublishedVersions" >
+          DependsOnTargets="PublishCoreHostPackagesToFeed;FinalizeBuildInAzure;UpdateVersionsRepo" />
+
+  <Target Name="UpdateVersionsRepo"
+          Condition="'$(GitHubAuthToken)' != ''"
```

Comment:
Do we want to standardize on either triggering this skip on one particular missing variable (`GitHubAuthToken`) or skipping on any missing variable (`DebRepoId` etc below)?  `GitHubUser` and `GitHubEmail` are also required for this, correct?